content/blog/2021-08-17-august-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,260 @@
+> you need in order to replace a pattern (if it is even matchable with regexes).
+
+> Every refactoring rule offers a single entrypoint, match(), where they accept
+> an AST node (from the ast module in the standard library) and respond with
+> either returning an action to refactor or nothing. If the rule succeeds on the
+> input, then the returned action will build a replacement node and refactor
+> will simply replace the code segment that belong to the input with the new
+> version.
+
+Way to go Batuhan! 🚀
+
+## July Office Hour Meetup
+
+If you missed our July Office Hours, good news! It's now available on our
+[YouTube Channel](https://www.youtube.com/channel/UC37rp97Go-xIX3aNFVHhXfQ) and
+you can see [João Santiago](https://twitter.com/jcpsantiago) shares about
+{dvthis}, and how his team at [Billie.io](https://www.billie.io/) uses DVC to
+productionize rstats.
+
+Also in the Meetup is a DVC Studio demo by
+[Tapa Dipti Situala](https://www.linkedin.com/in/tapa-dipti-sitaula/), Senior
+Product Engineer for Studio. You can catch the presentations along with great
+questions and discussion from the Community!
+
+https://www.youtube.com/watch?v=H22j1lWIvMw&t=1546s
+
+## Next Meetup
+
+So remember when I told you last month about DVC + Streamlit = ❤️ ? Well at our
+August Office Hours Meetup,
+[Antoine Toubhans](https://www.linkedin.com/in/antoine-toubhans-92262119/) of
+[Sicara](https://www.sicara.fr/) will be presenting
+[his tutorial](https://www.sicara.ai/blog/dvc-streamlit-webui-ml) on how to do
+just that! Join us in the integrating fun on August 19th at 3:00 pm UTC! RSVP at
+this link below! 👇🏼
+
+<external-link
+href="https://www.meetup.com/DVC-Community-Virtual-Meetups/events/279723437/"
+title="DVC Office Hours - DVC and Streamlit Integration"
+description="Antoine Toubhans of Sicara shares his tutorial for using Streamlit with DVC to create a customizable web UI"
+link="https://meetup.com"
+image="/uploads/images/2021-08-17/streamlit-oh.png"/>
+
+## Learning Opportunities
+
+This week's DVC Learn Meetup (August 18th) will be the last in our series of DVC
+Learn Meetups designed to get teams up and running with DVC. We will digest our
+learnings from this first cohort and revamp for the next set of three classes
+that will begin in September. Subscribe to
+[our Meetup group](https://www.meetup.com/DVC-Community-Virtual-Meetups/) and
+and follow us in [Twitter](https://twitter.com/DVCorg) and
+[LinkedIn](https://www.linkedin.com/company/18657719) to stay in the know about
+all of our upcoming events!
+
+If you are interested in weighing in on what kinds of educational content you
+would like to see from us, we'd be grateful if you'd fill out
+[**this survey**](https://docs.google.com/forms/d/e/1FAIpQLSdmwjs0ZkxDdODfZTvSwP2bVW4JAVVdxiYhQPyW5dSbsZC8qg/viewform?pli=1)
+to help us plan! 🙏🏼
+
+![DVC Online Course survey](/uploads/images/2021/08-17/survey.png) _Help us plan
```

Comment:
And this one.

Reply:
But I see they both render properly in https://dvc-org-august-heartbea-moy8x7.herokuapp.com/blog/august-21-dvc-heartbeat so yeah we're good to merge here.

Reply:
The actual URL rendered for this one is

/static/83fdf48f7311c67c558afe07fa5a639b/fa73e/survey.webp

Reply:
BTW what is this /uploads -> /static change? I suspect @rogermparent and/or @casperdcl may know. We need to update the link checker to support this.

Reply:
> BTW what is this /uploads -> /static thingy? I suspect @rogermparent and/or @casperdcl may know.

I believe this is due to Gatsby image processing, since the resulting imageset isn't at the same path as the original.

> We need to update the link checker to support or ignore this.

Yeah, probably, ignoring all MD image URLs would be the most obvious solution, and I don't think there doing so would clobber anything since we process all MD images now.

Reply:
https://github.com/iterative/link-check/issues/9